packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -73,7 +74,9 @@ export default class GoogleCalendarService implements Calendar {
   }
 
   private googleAuth = (credential: CredentialPayload) => {
-    const googleCredentials = googleCredentialSchema.parse(credential.key);
```

Comment:
I believe we don't need this parsing step as we already parse the token response using this schema or the credential syncing schema.